scripts/remove_operator.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+import subprocess
+import json
+import logging
+import os
+
+
+log = logging.getLogger(__name__)
```

Comment:
By default logging level set in Warn (if i'm not wrong), so those consequent `log.info` invocations will not cause log file update.
```suggestion
logging.basicConfig(level=logging.INFO)
```

Reply:
Thanks, added that